cli/tests/test_obslog_command.rs

```diff
@@ -142,6 +142,38 @@ fn test_obslog_with_or_without_diff() {
     "###);
 }
 
+#[test]
+fn test_obslog_with_custom_symbols() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+
+    std::fs::write(repo_path.join("file1"), "foo\n").unwrap();
+    test_env.jj_cmd_ok(&repo_path, &["new", "-m", "my description"]);
+    std::fs::write(repo_path.join("file1"), "foo\nbar\n").unwrap();
+    std::fs::write(repo_path.join("file2"), "foo\n").unwrap();
+    test_env.jj_cmd_ok(&repo_path, &["rebase", "-r", "@", "-d", "root()"]);
+    std::fs::write(repo_path.join("file1"), "resolved\n").unwrap();
+
+    let toml = concat!(
+        "templates.log_node_elided = true\n",
+        "templates.log_node = 'if(current_working_copy, \"$\", if(root, \"┴\", \"┝\"))'\n",
```

Comment:
nit: don't include a case for the root since it's not used

Reply:
Done.